core/lib/spree/testing_support/common_rake.rb

```diff
@@ -13,12 +9,38 @@ def initialize
     namespace :common do
       task :test_app, :user_class do |_t, args|
         args.with_defaults(user_class: "Spree::LegacyUser")
-        require ENV['LIB_NAME']
+        lib_name = ENV['LIB_NAME'] or
+          raise "Please provide a library name via the LIB_NAME environment variable."
+
+        require lib_name
 
         force_rails_environment_to_test
 
-        Spree::DummyGenerator.start ["--lib_name=#{ENV['LIB_NAME']}", "--quiet"]
-        Solidus::InstallGenerator.start ["--lib_name=#{ENV['LIB_NAME']}", "--auto-accept", "--with-authentication=false", "--payment-method=none", "--migrate=false", "--seed=false", "--sample=false", "--quiet", "--user_class=#{args[:user_class]}"]
+        Spree::DummyGenerator.start [
+          "--lib-name=#{lib_name}",
+          "--quiet",
+        ]
+
+        # After the dummy app is generated the working directory
+        # is changed to that of the dummy app.
```

Comment:
I'm not sure I understand that. It looks like I'm still in the directory from where I run the command (which is desirable). 🤔

Reply:
Yeah I was bitten by this in the past and so I though it was good to (at least) warn my future self, here's a demonstration:

```
solidus_paypal_commerce_platform:master * ⤑ bin/rake clobber extension:test_app --trace
To use retry middleware with Faraday v2.0+, install `faraday-retry` gem
** Invoke clobber (first_time)
** Invoke clean (first_time)
** Execute clean
** Execute clobber
rm -r /Users/elia/Code/Nebulab/solidus_paypal_commerce_platform/spec/dummy
** Invoke extension:test_app (first_time)
** Execute extension:test_app
** Invoke common:test_app (first_time)
** Execute common:test_app
Generating dummy Rails application...
      create  config/master.key

From: /Users/elia/Code/Nebulab/solidus/core/lib/spree/testing_support/common_rake.rb @ line 23 :

    18: 
    19:         Spree::DummyGenerator.start [
    20:           "--lib-name=#{lib_name}",
    21:           "--quiet",
    22:         ]
 => 23:         binding.irb
    24: 
    25:         # After the dummy app is generated the working directory
    26:         # is changed to that of the dummy app.
    27:         sh({
    28:           'SKIP_SOLIDUS_BOLT' => '1',

>> Dir.pwd
=> "/Users/elia/Code/Nebulab/solidus_paypal_commerce_platform/spec/dummy"
>> 
```

Reply:
Oh, I see. Would it be better to say "While the dummy app is generated" instead of "After the dummy app is generated"? I mean, when I run the task from the CLI, my directory is not changed after the task has run.